### PR TITLE
ADEN-7814 Disable INVISIBLE_SKIN call on non-Roadblock pages

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/provider/gpt/sraHelper.js [hunk 1]
```diff
@@ -10,8 +10,15 @@ define('ext.wikia.adEngine.provider.gpt.sraHelper', [
 		sraSlots = [
 			'INVISIBLE_SKIN',
 			'TOP_LEADERBOARD'
+		],
+		optionalSlots = [
+			'INVISIBLE_SKIN'
 		];
 
+	function isOptional(slotName) {
```
Comment (jacekbj) [inline extensions/wikia/AdEngine/js/provider/gpt/sraHelper.js]: Please add a description of what it means that a slot is optional.

Comment (dmnsobczak) [reply extensions/wikia/AdEngine/js/provider/gpt/sraHelper.js]: Those are the slots used only in UAP:Roadblock template. I renamed some parts of code to make it clear.

**Comment by fraszczakszymon** [comment]
What would happen if we simple disable `INVISIBLE_SKIN` in [slotsContext](https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/context/slotsContext.js#L38)? Wouldn't it be disabled by default? Roadblock is going to re-enable this slot for its purposes again

**Comment by dmnsobczak** [comment]
No, it's not gonna disable slot - we still have INVISIBLE_SKIN requests on non-Roadblock pages.

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**